WcaOnRails/app/models/post.rb

```diff
@@ -48,17 +48,23 @@ def self.search(query, params: {})
     posts.order(created_at: :desc)
   end
 
+  def url
+    Rails.application.routes.url_helpers.post_path(slug)
+  end
+
+  alias_attribute :createdAt, :created_at
```

Comment:
I did this only to keep the original behavior which set the `createdAt` attribute in the json, I totally agree with you that it's weird!
I unfortunately have no idea why it was done this way!

> (doesn't our Rails -> JS bridge hook convert everything to camelCase anyways??)

This I don't know, but it would be awesome if it did!
Taking a look at [the documentation](https://github.com/reactjs/react-rails#camelize-props) it looks like it can do it when passing props to the helper, but some (most?) our json object are actually loaded dynamically.
It would be awesome to have to automatic conversion so that we can: expect/emit snake_case attribute in the rails world, receive/send camelCase attribute in the js world!

Reply:
Ah, I know where my thought was coming from: We currently do some snake -> camel conversion manually.
For example: https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/app/webpacker/lib/wca-data/events.js.erb#L11 or https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/app/webpacker/lib/wca-data/formats.js.erb (and basically almost everything else in the `webpacker/lib/wca-data` folder...)

Reply:
Ah right.
I must admit it would be cool and shouldn't be too hard to do something like this when we fetch/post data to the website (maybe with a dedicated method so that the user knows the data attributes are being rewritten)!

Reply:
As to 

> I did this only to keep the original behavior which set the createdAt attribute in the json, I totally agree with you that it's weird!
I unfortunately have no idea why it was done this way!

See probably https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/app/webpacker/components/PostsWidget.js#L69

Reply:
> I must admit it would be cool and shouldn't be too hard to do something like this when we fetch/post data to the website (maybe with a dedicated method so that the user knows the data attributes are being rewritten)!

Thanks for spotting that! Although arguably it will not help with the current `wca-data` approach of injecting `erb` directly into the Webpacker code. I'll keep this issue on our radar and see if someone in WST is interested in picking that up to come up with a more consistent solution (especially since we desperately have to get rid of the `rails-erb-loader` anyways and load the data via "proper" APIs)

Reply:
Do you have an issue for tracking the rails erb loader thing?
I think there is actually a simpler solution, without having to load anything from APIs (because at the end of the day it's only static data): instead of hooking into webpacker we could simply generate the appropriate js/json file on demand, quite like how translations are generated, and use it when compiling js assets.

Reply:
No issue yet. But let's pause the discussion here and wait until that Issue has been created, I think that's a better place for exchanging arguments.

For now, let's focus on the `createdAt` vs. `created_at` issue. Would you mind changing the Posts component to either camelize first (or use the snake case field) or is that out of the scope of this PR?

Reply:
My initial thought was that it could break external stuff but looking back at why it's like this (oops...) it looks like it was not there in the far past so I guess it's fine to change it; I've updated the PR!